src/com/pentalog/pentastagiu/homework2/PR3/Board.java

```diff
@@ -0,0 +1,208 @@
+package com.pentalog.pentastagiu.homework2.PR3;
+
+
+import java.util.Scanner;
+
+public class Board {
+    ///this is gonna be the main app
+    static User[] users = new User[50];/// we are gonna have some users that are already on the server
```

Comment:
deocamdata e ok asa pt ca atunci cand ai implementat tema nu vorbisem inca despre colectii, dar atunci cand lurezi cu un numar necunoscut/dinamic de elemente intr-o colectie, e preferabil sa lucrezi cu un ArrayList(care in spate se foloseste tot de array-uri, dar are un mecanism mai eficient de alocare a memoriei)